src/passes/analysis/verilog.cpp

```diff
@@ -1130,12 +1130,9 @@ Passes::Verilog::compileModuleBody(
   for (auto instance : instances) {
     Module* instance_module = instance.second->getModuleRef();
     std::string module_name = instance_module->getLongName();
-    if (instance_module->isGenerated()) {
-      if (instance_module->getGenerator()->getMetaData().count("verilog") > 0) {
-        json verilog_json = instance_module->getGenerator()
-                              ->getMetaData()["verilog"];
-        module_name = make_name(instance_module->getName(), verilog_json);
-      }
+    if (instance_module->isGenerated() && instance_module->getGenerator()->getMetaData().count("verilog") > 0) {
```

Comment:
I change this line of code into
```
if (instance_module->isGenerated() && instance_module->getMetaData().count("verilog") > 0)
```

Reply:
@rdaly525 I think this means you need to check two separate cases here:
1. the instance_module has `"verilog"` metadata
2. the instance_module->getGenerator() has `"verilog"` metadata